src/query_parser/v2/parser.y

```diff
@@ -504,7 +511,22 @@ attribute(A) ::= ATTRIBUTE(B) COLON param_term(C). {
       value_len = found_value_len;
     }
   }
-  A = (QueryAttribute){ .name = B.s, .namelen = B.len, .value = value, .vallen = value_len };
+  A = (QueryAttribute){ .name = rm_strndup(B.s, B.len), .namelen = B.len, .value = value, .vallen = value_len };
+}
+
+attribute(A) ::= ATTRIBUTE(B) COLON STOPWORD(C). {
```

Comment:
as above, why differentiate between term and stopword?

Reply:
They are different tokens